great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -1181,9 +1190,7 @@ def _build_selectable_from_batch_spec(
                 sampler_fn = self._data_sampler.get_sampler_method(sampling_method)
                 return (
                     sa.select("*")
-                    .select_from(
-                        sa.table(table_name, schema=batch_spec.get("schema_name", None))
-                    )
+                    .select_from(subselectable)
```

Comment:
@billdirks While not part of this effort, it would be great to investigate the feasibility of using the formal table columns in the `sa.select("*")` statement (analogous to the way it is done as part of introspection functions in "great_expectations/expectations/metrics/util.py") -- the advantage is that this will provide objects for columns, which could then be utilized for other operations (e.g., adding labels, etc.), which are more difficult (if not outright impossible) when using the string `"*"`.  Thanks.

Reply:
@billdirks All I can think about is if the example(s) from `great_expectations/expectations/metrics/util.py:column_reflection_fallback()` and/or from `great_expectations/expectations/metrics/multicolumn_map_metrics/compound_columns_unique.py` and/or `great_expectations/execution_engine/sqlalchemy_execution_engine.py::ExecutionEngine.get_domain_records()` apply, then there may be a way; otherwise, we keep the `sa.select("*")` -- actually it looks like we resort to `sa.select([sa.text("*")])` as the catch-all in the latter.  So, perhaps worth a quick check, just in case.  Thanks!